app/ide-desktop/lib/dashboard/README.md

```diff
@@ -31,3 +31,39 @@ management, project sharing, and user accounts and authentication.
   certain static hosting providers.
 - `esbuild-config.ts`: Configuration for ESBuild based on the environment
   variables. This is a dependency of `esbuild-config.ts` in sibling modules.
+
+## Cloud environment variables
+
+These are environment variables related to the cloud backend. If these variables
+are not set, the build will still work, however access to the cloud backend will
+be disabled.
+
+Note that `ENSO_CLOUD_ENV` may be set to instead load the files from a `.env`
```

Comment:
Why is it `ENSO_CLOUD_ENV` in some places but `ENSO_CLOUD_ENVIRONMENT` in others? Do they have different purposes? If they do, perhaps we could consider more descriptive names?

Reply:
`ENSO_CLOUD_ENV` is the (somewhat legacy) variable that's used to define the entire set of `ENSO_CLOUD_` variable names. definitely agree that it's not particularly descriptive, i just decided to keep it like this so existing commands still work. i'll see what i can do though, it's definitely better to fix this now rather than later

Reply:
hmm... i fixed it but it's unreasonably verbose now. i'll see whether i can switch it to use vite's `--mode` parameter instead.

Reply:
seems like that's a no. the `mode` variable isn't accessible at the top level of the build script. instead i've merged it with `ENSO_CLOUD_ENVIRONMENT`.